lib/Kaba.js

```diff
@@ -18,6 +19,7 @@ const typeScriptErrorFormatter = require("@becklyn/typescript-error-formatter");
  *      js: {
  *          javaScriptDependenciesFileName: string,
  *          webpack: webpack.Configuration,
+ *          ?tsConfigFile: string,
```

Comment:
I would name this parameter like your property.

Also: is this valid syntax in JSDoc?